### ISSUE TITLE
Deprecation Cop's Listed Calls/Selectors Overlapping Text

### ISSUE THREAD

**Issue by sos12** [atom/deprecation-cop#24]
![depcopoverlap](https://cloud.githubusercontent.com/assets/6315034/5890301/4284aa10-a412-11e4-91cf-1a8485bc6a4c.gif)

As you can see, the Deprecation Cop is working for me, but its results are "blending". For clarity, I'm not wondering what I have to do to make the errors go away. I can fix the calls, and an update to the Linter removed the issue there. I'm pointing out the formatting of the error messages themselves.

Atom 0.175.0
OSX Yosemite 10.10.1
*Both are latest versions at time of post

**Comment by kevinsawicki** [comment]
Having some trouble reproducing this, can you reproduce it when running `atom --safe`?

**Comment by kevinsawicki** [comment]
Sorry, `atom --safe` won't exhibit it since packages won't load and the view will be empty.

Just to confirm, you are using the built-in themes? And do you have any custom styles in your `~/.atom/styles.less` file?

**Comment by sos12** [comment]
I don't remember putting any changes in styles.less, but as a precaution I
copy/pasted the file from the repository just so that could be ruled out.
Not exactly sure where the edits I had came from, but once the defaults
were in place the issue was resolved. UE somewhere along the line during my
customizations, I'm sure.

Apologies, and thanks for your great work on this great tool!

Skyler

On Mon, Feb 2, 2015 at 6:06 PM, Kevin Sawicki <notifications@github.com>
wrote:

> Sorry, atom --safe won't exhibit it since packages won't load and the
> view will be empty.
>
> Just to confirm, you are using the built-in themes? And do you have any
> custom styles in your ~/.atom/styles.less file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/deprecation-cop/issues/24#issuecomment-72572751>.
>



-- 

Skyler Osborn

801-616-1515
skyler.osborn@gmail.com
